docs/reference/mapping/types/range.asciidoc

```diff
@@ -229,3 +235,208 @@ The following parameters are accepted by range types:
     Whether the field value should be stored and retrievable separately from
     the <<mapping-source-field,`_source`>> field. Accepts `true` or `false`
     (default).
+
+[[range-synthetic-source]]
+==== Synthetic `_source`
+
+IMPORTANT: Synthetic `_source` is Generally Available only for TSDB indices
+(indices that have `index.mode` set to `time_series`). For other indices
+synthetic `_source` is in technical preview. Features in technical preview may
+be changed or removed in a future release. Elastic will work to fix
+any issues, but features in technical preview are not subject to the support SLA
+of official GA features.
+
+`range` fields support <<synthetic-source,synthetic `_source`>> in their default
+configuration. Synthetic `_source` cannot be used together with
+<<copy-to,`copy_to`>> or with <<doc-values,`doc_values`>> disabled.
```

Comment:
This mentions `copy-to` because other field types have that restriction and i have added it in code too for consistency. `copy-to` is not documented for ranges though and is not really useful. You can create mapping with `copy-to` provided but all operations will fail because ranges are objects and copy to for them is not supported. I wonder if we should disable `copy-to` support for ranges?

Reply:
Okay, there is one use case when `copy-to` works - when using `ip_range` specifically with CIDR strings.

Reply:
Let me remove mention of `copy_to` from here given that it is not mentioned anywhere to begin with. I'll open an issue for the decision to disable it completely for ranges.

Reply:
#107445